modules/tell.py

```diff
@@ -11,33 +11,41 @@
 """
 
 import os, re, time, random
-import web
+import threading
 
 maximum = 4
 
 
-def loadReminders(fn):
-    result = {}
-    f = open(fn)
-    for line in f:
-        line = line.strip()
-        if line:
-            try: tellee, teller, verb, timenow, msg = line.split('\t', 4)
-            except ValueError: continue  # @@ hmm
-            result.setdefault(tellee, []).append((teller, verb, timenow, msg))
-    f.close()
+def loadReminders(fn, lock):
+    lock.acquire()
+    try:
+        result = {}
+        f = open(fn)
+        for line in f:
+            line = line.strip()
+            if line:
+                try: tellee, teller, verb, timenow, msg = line.split('\t', 4)
+                except ValueError: continue  # @@ hmm
+                result.setdefault(tellee, []).append((teller, verb, timenow, msg))
```

Comment:
If this blows up in the except above the values appended will be None type, does that screw anything up?